pagopa/io-app
# [IOBP-518, IOBP-519] Handle payment dismissal after authorization request

Depends on https://github.com/pagopa/io-app/pull/5419

## Short description
This PR add the handling of payment dismissal after payment authorization request

## List of changes proposed in this pull request
- Added `onDismiss` in `useWalletPaymentAuthorizationModal` hook
- Added outcome screen navigation on authorization dismissal

## How to test
Start a new payment flow from **Profile > Playgrounds > New Wallet > Payment**, dismiss the authorization webview in the last step and check that the payment is correctly cancelled.

## Previews

https://github.com/pagopa/io-app/assets/6160324/09a9ee38-3134-4053-8b4b-e5b6aa08b6ee
